crates/fluvio-spu/src/kv/consumer.rs

```diff
@@ -169,6 +169,29 @@ impl SharableConsumerOffsetStorage {
         self.0.write().await.delete(key).await
     }
 
+    pub async fn delete_by_replica_key(&self, replica: &ReplicaKey) -> Result<()> {
```

Comment:
this doesn't need to be public.
add tracing

Reply:
sure, done! it's pub(crate) now